src/libwalletqt/WalletManager.cpp

```diff
@@ -183,6 +198,16 @@ bool WalletManager::addressValid(const QString &address, bool testnet) const
     return Monero::Wallet::addressValid(address.toStdString(), testnet);
 }
 
+bool WalletManager::keyValid(const QString &key, const QString &address, bool isViewKey,  bool testnet) const
+{
+    std::string error;
+    if(!Monero::Wallet::keyValid(key.toStdString(), address.toStdString(), isViewKey, testnet, error)){
+        qDebug() << QString::fromStdString(error);
```

Comment:
Not out of the box. I tried to include contrib/epee/include/misc_log_ex.h but got build errors from boost. I'll investigate it further outside this PR.